elyra/metadata/schemas/airflow.json

```diff
@@ -89,6 +89,15 @@
             "placeholder": "https://your-cos-service:port"
           }
         },
+        "cos_secret": {
+          "title": "Cloud Object Storage Secret",
+          "description": "Name of the secret stored on the kubernetes host containing the Username and Password",
+          "type": "string",
+          "uihints": {
+            "secure": true,
+            "category": "Cloud Object Storage"
```

Comment:
While it makes sense to associate it with the `Cloud Object Storage` category, there's going to be a need to support Kubernetes settings that are not part of any of the existing category. (Volume mounts come to mind.) Therefore I would suggest to add this property to a new `Kubernetes` category.

Reply:
I also believe that this puts a stake in the ground, which establishes a Kubernetes settings hierarchy:
- Settings that apply to all pipelines/batch executions that use a specific runtime.
- Runtime settings can optionally be overridden by pipeline settings.
- Pipeline settings can optionally be overridden by node settings.